docs/src/modules/components/MarkdownDocs.js

```diff
@@ -55,6 +55,32 @@ export default function MarkdownDocs(props) {
   const Provider = isJoy ? CssVarsProvider : React.Fragment;
   const Wrapper = isJoy ? BrandingProvider : React.Fragment;
 
+  const directTsDemos = rendered
```

Comment:
I think that we should have a `if (process.env.NODE_ENV !== 'production') {` to remove the bundle size in production, this error is for the DX in dev mode.